core/windows_and_tabs/window_snap.py

```diff
@@ -302,14 +328,170 @@ def __init__(self, left, top, right, bottom):
 }
 
 
+def _snap_next(windows: list[Any], target_layout: RelativeScreenPos) -> Optional[int]:
+    print(windows)
+    for index, window in enumerate(windows):
+        if window is None:
+            return index
+        try:
+            _snap_window_helper(
+                window,
+                target_layout,
+            )
+            return index
+        except Exception as e:
+            print(f"Failed to snap window: {e}")
```

Comment:
Would recommend you use notifications rather than just a log message to identify errors to the user.

Reply:
As many applications have hidden windows this is not really an error case-not in any significant sense. The zoom application for example always has a hidden window in front that cannot be snapped which would cause this all the time.

Reply:
If the consensus is that this is a better thing to notify for than I'm happy to do it, though I expect that most people will be notified almost every time they run the command-sometimes even multiple times. I'm happy to handle this however is recommended.

Reply:
Yeah, if this is an expected case for invisible windows that will happen frequently, I would change the print statement to be less scary (if we have it at all). I would also put the application name and window title in the message to see what it was.

Reply:
Sure I can do that